packages/server-wallet/src/wallet/store.ts

```diff
@@ -357,6 +361,12 @@ export class Store {
     return (await Channel.query(this.knex)).map(channel => channel.protocolState);
   }
 
+  async getLedgerChannels(): Promise<ChannelState[]> {
+    return (await Channel.query(this.knex).whereNotNull('assetHolderAddress')).map(
+      channel => channel.protocolState
+    );
+  }
```

Comment:
Should perhaps live in the `Channel` model/class?

Reply:
02a9216dd